src/ec/google/chromeec/ec_sync.c

```diff
@@ -25,6 +25,11 @@
 #include <inttypes.h>
 #include <stddef.h>
 
+#if CONFIG(CHROMEEC_AFTER_G3_STATE)
```

Comment:
I'm not sure any of this needs to be guarded using the preprocessor. If the enums align, you don't need to include pmclib, and you can guard the call with `if (CONFIG(CHROMEEC_AFTER_G3_STATE))` rather than the preprocessor